### ISSUE TITLE
NTR: Glomerular Sclerosis (level 3)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6262]
Glomerular Sclerosis (level 3)
parent:Obliterative abnormalities of the renal glomerulus ()
def: @avizrosenberg @yongqunh

**Comment by pnrobinson** [comment]
@avizrosenberg @yongqunh

I think that this is a synonym of this existing term, correct?
https://hpo.jax.org/app/browse/term/HP:0000096  (Glomerulosclerosis)

Should we use this term and change the primary label to ``Glomerular sclerosis``

**Comment by pnrobinson** [comment]
I am going to add this as a synonym, please reopen if this is incorrect. @avizrosenberg @yongqunh
